prototype2/common/DataSave.cpp

```diff
@@ -33,14 +35,7 @@ DataSave::DataSave(std::string filename) : filename_prefix(filename) {
 DataSave::DataSave(std::string name, uint64_t maxlen)
     : filename_prefix(name), maxfilesize(maxlen) {
 
-  char cStartTime[50];
-  time_t rawtime;
-  struct tm *timeinfo;
-  time(&rawtime);
-  timeinfo = localtime(&rawtime);
-
-  strftime(cStartTime, 50, "%Y%m%d-%H%M%S", timeinfo);
```

Comment:
Why change the filename from Ymd-HMS to Y-m-d-h-m-s ?

Its fine to refactor, but is there a reason to change the behaviour?

Reply:
No, I think I just didn't notice the difference. The two implementations we had around were probably different and I picked one at random? I don't care. I can make it whatever you prefer.

Reply:
Done